meta.yaml

```diff
@@ -104,8 +109,11 @@ tech:
     name: autotrack
     url: https://github.com/googleanalytics/autotrack/
   core_reporting_api:
-    name: Core Reporting API
+    name: Analytics Reporting API
     url: https://developers.google.com/analytics/devguides/reporting/core/
```

Comment:
Should we keep this as Core Reporting API v3 ?
This URL will redirect to latest version which is v4 so I suggest changing this to /core/v3. Optionally you may want to change URL below to just /core/